examples/open_source_models/role_playing_with_llama2.py

```diff
@@ -0,0 +1,90 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from colorama import Fore
+
+from camel.configs import OpenSourceConfig
+from camel.societies import RolePlaying
+from camel.typing import ModelType
+from camel.utils import print_text_animated
+
+
+def main(model_type=None) -> None:
```

Comment:
Let's put `model_path` and `model_path`here. And reduce the examples to one
```suggestion
def main(model_type=None, model_path=" ", serverl_url=" ") -> None:
```